bluesky/run_engine.py

```diff
@@ -1591,6 +1592,27 @@ def _save(self, msg):
         self.log.debug("Emitted Event with data keys %r (uid=%r)", data.keys(),
                        event_uid)
 
+    @asyncio.coroutine
+    def _drop(self, msg):
+        """Drop the event that is currently being bundled
+
+        Expected message object is:
+
+            Msg('drop')
+        """
+        if not self._bundling:
+            raise IllegalMessageSequence("A 'create' message must be sent, to "
+                                         "open an event bundle, before that "
+                                         "bundle can be dropped with 'drop'.")
+        if not self._run_is_open:
```

Comment:
I am not sure if we can actually reach this code path.  I think you have to have a run open to enter the bundling state so if `self._bundling` is True and `self._run_is_open` is False we have much bigger problems.

Reply:
I think you are correct. I included this unreachable code path because it is also included in `save`, and I wanted to make the control flow logic for `drop` and `save` as similar as possible.